cfgov/core/management/commands/inactive_users.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import unicode_literals
+
+from datetime import timedelta
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.core import mail
+from django.db.models import Q
+from django.utils import timezone
+from django.utils.formats import date_format
+
+
+class Command(BaseCommand):
+    help = 'Find users who have been inactive for a given amount of time'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--period',
+            type=int,
+            default=90,
+            help='The number of days that defines inactivity'
+        )
+        parser.add_argument(
+            '--emails',
+            nargs='+',
+            default=[],
+            help='Email output to a list of addresses'
+        )
+
+    def handle(self, *args, **options):
+        period = options['period']
+        emails = options['emails']
+
+        last_possible_date = timezone.now() - timedelta(days=period)
+
+        User = get_user_model()
+        inactive_users = User.objects.filter(
+            Q(is_active=True),
+            Q(date_joined__lt=last_possible_date),
```

Comment:
I did it because otherwise I was passing keyword arguments before non-keyword arguments because, logically, this is the order I was thinking about them in. But it occurs to me that doesn't matter for the query itself.